web/themes/joinup/joinup_theme.theme

```diff
@@ -641,6 +641,23 @@ function joinup_theme_preprocess_field_multiple_value_form(&$variables) {
 }
 
 /**
+ * Implements hook_preprocess_HOOK() for field--comma-separated.html.twig.
+ */
+function joinup_theme_preprocess_field__comma_separated(&$variables) {
+  // Only in listing modes.
+  $view_modes = [
+    'view_mode_tile',
+    'view_mode_featured',
+  ];
+
+  // Limit number of displayed items.
+  $items_limit = 3;
+  if ($variables['field_name'] == 'field_keywords' && in_array($variables['element']['#view_mode'], $view_modes)) {
```

Comment:
Unfortunately, the limitation settings don't apply:

![tags_limit](https://user-images.githubusercontent.com/1923476/30172070-05922d6a-93f4-11e7-8510-5b2afe36c617.gif)

This is with latest version of the branch + latest develop + `drupal cr all` which is reinforced during the demo.

The `items_limit` can be hoisted in the beginning of the function as any other variable definition.